cms/djangoapps/contentstore/views/course.py

```diff
@@ -1646,3 +1650,24 @@ def _get_course_creator_status(user):
         course_creator_status = 'granted'
 
     return course_creator_status
+
+
+def _get_library_creator_status(user):
+    """
+    Helper method for returning the library creation status for a particular user,
+    taking into account the values of DISABLE_LIBRARY_CREATION and LIBRARIES_ENABLED.
+
+    """
+
+    if not LIBRARIES_ENABLED:
+        library_creator_status = 'disallowed_for_this_site'
```

Comment:
IMHO returning a bool would be nicer since there are only two outcomes from this method.